ext/spl/tests/gh14687.phpt

```diff
@@ -0,0 +1,44 @@
+--TEST--
+GH-14687 segfault on debugging SplObjectStorage instance after __destruct.
+--CREDITS--
+YuanchengJiang
+--EXTENSIONS--
+phar
+--INI--
+phar.require_hash=0
+phar.readonly=0
+--FILE--
+<?php
+$fname = __DIR__ . '/' . basename(__FILE__, '.php') . '.phar.zip';
```

Comment:
You're creating a file here, but have no clean section. Please add a clean section.

Reply:
Also it's better to just hardcode the filename to `__DIR__ . '/gh14687.phar.zip'` because otherwise you can't use the same name in the clean section because the base name for the clean file contains the "clean" word.